src/test/java/bowling/domain/FinalFrameTest.java

```diff
@@ -0,0 +1,107 @@
+package bowling.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FinalFrameTest {
+
+    @DisplayName("bowl test")
+    @Test
+    void bowl() {
+        Frame frame = new FinalFrame();
+
+        frame.bowl(Pins.of(4));
+
+        assertThat(frame.getFirstPin()).isEqualTo(Pins.of(4));
+    }
+
+    @DisplayName("안쳤을 때는 False, 한번 만 첬을 때는 False, 두번 다 쳤을 때는 True 테스트")
+    @Test
+    void isNormalFinish(){
```

Comment:
```suggestion
    void isNormalFinish() {
```

매의 눈 @_@

IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.

Reply:
아이구 ...! 감사합니다 ㅎㅎ